gr-qtgui/lib/eye_sink_f_impl.h

```diff
@@ -0,0 +1,148 @@
+/* -*- c++ -*- */
+/*
+ * Copyright 2011-2013,2015,2020 Free Software Foundation, Inc.
```

Comment:
Since this is a new file, the copyright should be `2020` 

Please double check other files, if they are new and not updated or modified from older versions they should only have the current year.